README.md

```diff
@@ -0,0 +1,71 @@
+## krabby
+a code conformance module runner.
+
+![edna](https://cloud.githubusercontent.com/assets/465414/7166524/3ceb63e2-e37c-11e4-8bde-528c6284e062.png)
+
+### what?
+krabby is an extremely unopinionated module runner, similar to [Gulp](http://gulpjs.com/) or [Grunt](http://gruntjs.com/), but rather than helping create your build, it grades it based on tests and rules that you define.
+
+You pick and choose the tests that you want to use to test your code, and another set of tests you use to grade the results. This gives you the ability to check your code's quality from the biggest of pictures down to the finest of detail.
+
+## why?
+When working with teams larger than just yourself, it is common for a number of different styles to emerge. Code that is totally fine with one person, may be completely unacceptable to another. As a result, it becomes harder and harder to be able to trust code that you yourself didn't write. This is where krabby shines! By creating your own `krabby.json` file, you can quickly get a detailed report on whether or not any piece of code is up to your standards.
+
+## how?
+Before anything else, you will need to install krabby.
+
+```
+npm install -g krabby
```

Comment:
yep. either a global install, or `./node_modules/.bin/krabby` with `--config` passed in.